### PR TITLE
Chef smart proxy plugin support

### PR DESCRIPTION
**Pull request by ares**
Follow up of https://github.com/theforeman/puppet-foreman_proxy/pull/152

### CODE REVIEW

File: config/foreman-installer.yaml [hunk 1]
```diff
@@ -105,6 +105,9 @@
     :manifest_name: plugin/abrt
     :params_name: plugin/abrt/params
     :dir_name: foreman_proxy
+  :foreman_proxy::plugin::chef:
+    :manifest_name: plugin/chef
+    :params_name: plugin/chef/params
```
Comment (mmoll) [inline config/foreman-installer.yaml:110]: dir_name?

**Comment by ares** [comment]
@mmoll thanks, fixed

**Comment by mmoll** [comment]
:+1:

**Comment by ekohl** [comment]
:+1:

**Comment by domcleal** [comment]
Thanks!

**Pull request merged**